syntax = "proto3";

option go_package = "keyem4251/try-grpc-microservice-auth/pb";

service  Greeter  {
    rpc SayHello (HelloRequest) returns (HelloResponse);
}

message HelloRequest {
    string name = 1;
}

message HelloResponse {
    string message = 1;
}
